pdns/reczones.cc

```diff
@@ -362,17 +362,16 @@ std::shared_ptr<SyncRes::domainmap_t> parseAuthAndForwards()
 
```

Comment:
These changes do not seem related, better leave them out.

Reply:
gone!